### PR TITLE
OSS prow: auto deploy at 11:30, skip if no oncall

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml [hunk 1]
```diff
@@ -354,7 +354,7 @@ periodics:
       secret:
         secretName: google-oss-robot-ssh-keys
         defaultMode: 0400
-- cron: "05 17 * * 1-5"  # Bump with label `skip-review`. Run at 10:05 PST (17:05 UTC) Mon-Fri
+- cron: "30 18 * * 1-5"  # Bump with label `skip-review`. Run at 11:30 PST (18:05 UTC) Mon-Fri
```
Comment (fejta) [inline prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml:357]: Isn't 1a a better time? Seems like this is more likely to deploy right when someone's going to head off to lunch.

**Review by fejta — APPROVED** [review]
/hold

File: prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml (hunk 1 above)
Comment (chaodaiG) [reply prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml]: good point and done

**Review by fejta — APPROVED** [review]

**Pull request merged**